src/Gui/View3DInventorViewer.cpp

```diff
@@ -2379,6 +2379,23 @@ void View3DInventorViewer::setSeekMode(SbBool on)
                                 NavigationStyle::IDLE : NavigationStyle::INTERACT));
 }
 
+SbVec3f View3DInventorViewer::getCenterPointOnFocalPlan() {
+    SoCamera* cam = getSoRenderManager()->getCamera();
+    if (!cam)
+        return false;
```

Comment:
Error here : `/home/ob/dev/FreeCAD/src/Gui/View3DInventorViewer.cpp:2385:16: error: no viable conversion from returned value of type 'bool' to function return type 'SbVec3f'`

Reply:
Sorry. Fixed it.

Reply:
* typo in getCenterPointOnFocalPlan => getCenterPointOnFocalPlane
* handle const-correctness => `SbVec3f View3DInventorViewer::getCenterPointOnFocalPlan() const`

Reply:
fixed